Podfile

```diff
@@ -89,7 +92,7 @@ target 'SessionUtilitiesKit' do
   pod 'PromiseKit', :inhibit_warnings => true
   pod 'PureLayout', '~> 3.1.8', :inhibit_warnings => true
   pod 'SAMKeychain', :inhibit_warnings => true
-  pod 'SignalCoreKit', git: 'https://github.com/signalapp/SignalCoreKit.git', :inhibit_warnings => true
+  pod 'SignalCoreKit', git: 'https://github.com/mpretty-cyro/SignalCoreKit.git', branch: 'session-version', :inhibit_warnings => true
   pod 'YapDatabase/SQLCipher', :git => 'https://github.com/loki-project/session-ios-yap-database.git', branch: 'signal-release', :inhibit_warnings => true
 end
 
```

Comment:
Could you ignore the pod modification in this PR?

Reply:
Yea I can do that - there will still be a minor change (needed to add `NVActivityIndicatorView` to the share extension).

I can only test locally with these changes included unfortunately so won't be able to confirm there aren't any build issues without them

Reply:
No worries. I'll have Emily to test this before we merge it to dev.